alchemist-graphql/build.gradle.kts

```diff
@@ -93,20 +87,41 @@ kotlin {
                 implementation(alchemist("test"))
             }
         }
-        val jsMain by getting {
-            dependencies {
-                implementation(libs.apollo.runtime)
-                implementation(libs.kotlin.coroutines.core)
-            }
-        }
-        val jsTest by getting
     }
 }
 
 application {
     mainClass.set("it.unibo.alchemist.Alchemist")
 }
 
+/**
+ * Configure the Apollo Gradle plugin to generate Kotlin models
+ * from the GraphQL schema inside the `commonMain` sourceSet.
+ */
+apollo {
+    service("alchemist") {
+        generateKotlinModels.set(true)
+        packageName.set("it.unibo.alchemist.boundary.graphql.client")
+        schemaFiles.from(file("src/commonMain/resources/graphql/schema.graphqls"))
+        srcDir("src/commonMain/resources/graphql")
+        outputDirConnection {
+            connectToKotlinSourceSet("commonMain")
```

Comment:
@AngeloFilaseta wouldn't it be better to declare a new `generatedCommonMain` sourceset? What do you think?

Reply:
If this is inevitable, separation can certainly help. I would also add some indication somewhere (a readme in the sourceset or something similar) to ensure that developers can tedit what's inside only if they are really sure of what they're doing.

Reply:
This should be fixed also, @AngeloFilaseta confirm and resolve in case